requirements.txt

```diff
@@ -49,23 +49,23 @@ docutils==0.15.2
 draftjs-sanitizer==1.0.0
 elasticsearch==6.3.1
 elasticsearch-dsl==6.0.1
-faker==2.0.3
+faker==2.0.4
 freezegun==0.3.12
 google-api-core==1.14.3
-google-auth==1.7.0
+google-auth==1.7.1
 google-cloud-core==1.0.3
-google-cloud-storage==1.22.0
+google-cloud-storage==1.23.0
 google-i18n-address==2.3.5
 google-measurement-protocol==1.0.0
-google-resumable-media==0.4.1
+google-resumable-media==0.5.0
 googleapis-common-protos==1.6.0
 gprof2dot==2016.10.13
-graphene==2.1.8
-graphene-django==2.6.0
+graphene==2.1.6
+graphene-django==2.5.0
 graphene-django-optimizer==0.6.1
 graphene-federation==0.0.3
 graphql-core==2.2.1
-graphql-relay==2.0.0
+graphql-relay==0.4.5
```

Comment:
Why are some versions degraded?

Reply:
I've been looking into this. the only way i could reproduce this from master was by running `poetry update` which is weird in itself. Also, running `poetry lock` gives a different set of versions. Not sure how poetry's dependency resolution works, but it seems inconsistent.

Anyways, I've just reverted the lock file and re-installed the packages. it seems to be ok now.